spartan/scripts/test_upgrade_rollup_with_lock.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# This script is used to test the upgrade rollup with lock.
+# The point of the test is primarily to ensure that the CLI is not broken, and the upgrade works.
+
+set -eu
+
+NAMESPACE=$1
```

Comment:
This is just a duplication of what we have in jest, I'd much rather we have one code path and run the upgrade_rollup_with_lock script from there